Flow.Launcher/ViewModel/ResultViewModel.cs

```diff
@@ -68,17 +68,20 @@ public ResultViewModel(Result result, Settings settings)
                         OnPropertyChanged(nameof(Image));
                     });
 
-                if (Result.Glyph.FontFamily.Contains('/'))
+                if (Result.Glyph is { FontFamily: not null } glyph)
                 {
-                    var fontPath = Result.Glyph.FontFamily;
-                    Glyph = Path.IsPathRooted(fontPath) ? Result.Glyph : Result.Glyph with
+                    if (glyph.FontFamily.Contains('/'))
```

Comment:
is this neccessary? can you get the path always and use it without needing to check if it's a path?

Reply:
It allows direct font name if it is installed in the computer. For example, windows 10+ includes Segoe MDL2 Asset so we may not need to bundle a font.

Reply:
This line is for checking if the string is a directory? If so is there a better way to check this?

Reply:
Yeah I don't know whether there's a better way to do this. Probably checking the extension name is also fine?

Reply:
just added a comment, we can work something out later like a dedicated function for determining if string is folder or file path